applications/ParticleMechanicsApplication/tests/cpp_tests/test_particle_math_utilities.cpp

```diff
@@ -136,25 +133,25 @@ namespace Testing
         Matrix A = CreateSymmetricTest3Matrix3x3();
         ParticleMechanicsMathUtilities<double>::QRFactorization(A, Q, R);
 
-        KRATOS_CHECK_LESS_EQUAL((-7.071068e-01 - Q(0,0))/Q(0,0), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 4.082483e-01 - Q(0,1))/Q(0,1), tolerance);
-        KRATOS_CHECK_LESS_EQUAL((-5.773503e-01 - Q(0,2))/Q(0,2), tolerance);
-        KRATOS_CHECK_LESS_EQUAL((-7.071068e-01 - Q(1,0))/Q(1,0), tolerance);
-        KRATOS_CHECK_LESS_EQUAL((-4.082483e-01 - Q(1,1))/Q(1,1), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 5.773503e-01 - Q(1,2))/Q(1,2), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 0.000000e+00 - Q(2,0))/Q(2,0), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 8.164966e-01 - Q(2,1))/Q(2,1), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 5.773503e-01 - Q(2,2))/Q(2,2), tolerance);
-
-        KRATOS_CHECK_LESS_EQUAL((-1.414214e+00 - R(0,0))/R(0,0), tolerance);
-        KRATOS_CHECK_LESS_EQUAL((-7.071068e-01 - R(0,1))/R(0,1), tolerance);
-        KRATOS_CHECK_LESS_EQUAL((-7.071068e-01 - R(0,2))/R(0,2), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 0.000000e+00 - R(1,0))/R(1,0), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 1.224745e+00 - R(1,1))/R(1,1), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 4.082483e-01 - R(1,2))/R(1,2), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 0.000000e+00 - R(2,0))/R(2,0), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 0.000000e+00 - R(2,1))/R(2,1), tolerance);
-        KRATOS_CHECK_LESS_EQUAL(( 1.154701e+00 - R(2,2))/R(2,2), tolerance);
+        KRATOS_CHECK_LESS_EQUAL((-7.071068e-01 - Q(0,0))/Q(0,0), 1e-6);
```

Comment:
FYI you could consider refactoring this to use the new check macros for matrix and vector
this should compile faster, since unrolling macros takes time